numpyro/handlers.py

```diff
@@ -94,7 +94,7 @@ def apply_stack(msg):
         if msg.get("stop"):
             break
     if msg['value'] is None:
-        msg['value'] = msg['fn'](*msg['args'], **msg['kwargs'])
+        msg['value'] = msg['fn'].rvs(*msg['args'], **msg['kwargs'])
```

Comment:
Is this change caused by the drop of scipy's patch?

Reply:
This is because the patch doesn't address frozen instances from multivariate distributions, and I didn't want to add to the `patch` hack. :)

Reply:
Understood! Will fix it too. :)